packages/babel-plugin-ember-test-metadata/get-normalized-file-path.js

```diff
@@ -1,36 +1,32 @@
-const {
-  _getRelativePathForClassic,
-  _getRelativePathForClassicInRepo,
-  _getRelativePathForEmbroider,
-  _getRelativePathForEmbroiderInRepo,
-} = require('./get-relative-paths');
-
-const path = require('path');
-
 /**
  * Get a normalized file path, based on whether the app build is classic or with Embroider
  * @param {object} opts Babel state.file.opts which include root and filename props
  * @returns {string} E.g. tests/acceptance/my-test.js
  */
-function getNormalizedFilePath({ packageName, isUsingEmbroider, filename, root }) {
-  if (!isUsingEmbroider) {
-    if (filename.includes('ember-add-in-repo-tests')) {
-      return _getRelativePathForClassicInRepo(filename);
-    }
+function getNormalizedFilePath({ packageName, filename, isUsingEmbroider, root, sourceFileName }) {
+  let file = sourceFileName || filename;
```

Comment:
there a step in some ember build processes that doubled the package path, e.g. `/path/to/project/extended/extended/tests/my-test.js`. `sourceFileName` returns `extended/tests/my-test.js` which is correct.